src/main/java/org/prebid/server/auction/requestfactory/VideoRequestFactory.java

```diff
@@ -111,6 +112,7 @@ public Future<WithPodErrors<AuctionContext>> fromRequest(RoutingContext routingC
                 .compose(httpRequest -> createBidRequest(httpRequest)
                         .map(bidRequest -> removeEmptyEids(bidRequest, initialAuctionContext.getDebugWarnings()))
                         .compose(bidRequest -> validateRequest(
+                                initialAuctionContext.getAccount(),
```

Comment:
`initialAuctionContext.getAccount()` will always return `null` here

Reply:
I think we can accept it this way, since we haven't supported Video endpoint for many months now.